scss/styles.scss

```diff
@@ -52,6 +52,23 @@
   }
 }
 
+.image-container {
+  position: relative;
+
+  &::before {
+    background: url("/src/img/badge.png");
+    background-repeat: no-repeat;
```

Comment:
you can add the img on the content to avoid all these background properties and to only specify a width or a height

Reply:
content: url("/src/img/badge.png");